examples/persistent-storage/cc13x2x7_26x2x7/BUILD.gn

```diff
@@ -28,22 +30,33 @@ project_dir = "${chip_root}/examples/persistent-storage/cc13x2x7_26x2x7"
 
 ti_simplelink_sdk("sdk") {
   include_dirs = [ "${project_dir}/include" ]
-
-  defines = []
-  if (is_debug) {
-    defines += [ "BUILD_RELEASE=0" ]
-  } else {
-    defines += [ "BUILD_RELEASE=1" ]
-  }
+  public_configs = [ ":sysconfig_config" ]
 }
 
 ti_sysconfig("sysconfig") {
   sources = [ "${project_dir}/chip.syscfg" ]
 
   outputs = [
-    "ti_devices_config.c",
+    "ti_radio_config.c",
```

Comment:
why does the kvs fix require adding radio/ble and such source files and headers?

I assume that this is unrelated but safe/needed?

Reply:
I was working through build errors and started copying from the working examples. I can pair them down if it is needed.